src/tiled/objectselectiontool.cpp

```diff
@@ -533,6 +534,42 @@ void ObjectSelectionTool::mousePressed(QGraphicsSceneMouseEvent *event)
 
         break;
     }
+    case Qt::RightButton:
+        if (event->modifiers() & Qt::AltModifier) {
+            QList<MapObjectItem*> underlyingObjects = objectItemsAt(event->scenePos());
+            if (underlyingObjects.empty())
+                break;
+            QMenu selectUnderlyingMenu;
+
+            for (int levelNum = 0; levelNum < underlyingObjects.size(); ++levelNum) {
+                const QString& objectName = underlyingObjects[levelNum]->mapObject()->name();
+                QString actionName = QLatin1String(levelNum < 9 ? "&" : "") + tr("%n) ", "", levelNum + 1)
```

Comment:
Is there any better way to construct a `"%d) "` string in this case? I think it doesn't require any translation, so can be made without `tr`.
And how to properly update a .ts file? I can provide a russian translation for strings in my code (now it is only "Unnamed object")